osu.Game/Screens/Play/PauseOverlay.cs

```diff
@@ -64,7 +64,7 @@ public override bool OnPressed(KeyBindingPressEvent<GlobalAction> e)
             switch (e.Action)
             {
                 case GlobalAction.PauseGameplay:
-                    BackAction.Invoke();
+                    InternalButtons.Children.First().TriggerClick();
```

Comment:
```suggestion
                    InternalButtons.First().TriggerClick();
```

I think this should work (and read better).